lib/spack/spack/test/data/modules/tcl/invalid_token_in_env_var_name.yaml

```diff
@@ -3,7 +3,7 @@ enable:
 tcl:
   all:
     filter:
-      environment_blacklist':
+      environment_blacklist:
```

Comment:
This reminds me that Hai Ah asked us to refactor these -- we need to get rid of whitelists and blacklists in spack for inclusive naming -- that can be in a separate PR.